### PR TITLE
Add "read" command to read and execute a SQL file

### PR DESCRIPTION
**Pull request by sco11morgan**
## Description

Add command to allow executing a query file while inside the REPL.

Heavily borrowed from `execute_favorite_query` and `watch_query` commands.

Resolves https://github.com/dbcli/athenacli/issues/60

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

**Comment by sco11morgan** [comment]
One part I'm stuck on is even though the query loads and executes as expected.  The prompt always ends with 
```Query is none or empty```

**Comment by zzl0** [comment]
Thanks for your PR, I will take a look today

**Comment by zzl0** [comment]
> One part I'm stuck on is even though the query loads and executes as expected. The prompt always ends with
Query is none or empty

If the last line in your file ends with `\n`, there will be a empty in the result of `sqlparse.split`, and the empty string cause the `Query is none or empty`  error message.

```python
>>> sqlparse.split("select * from a;\n")
['select * from a;', '']
```

File: athenacli/packages/special/iocommands.py [hunk 1]
```diff
@@ -219,6 +219,34 @@ def subst_favorite_query_args(query, args):
 
     return [query, None]
 
+@special_command('load', 'load [filename]', 'Load and execute query from a file.')
```
Comment (zzl0) [inline athenacli/packages/special/iocommands.py:222]: Do you mind renaming this to `read` (like the [sqlite3 .read command](https://sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_))? Just try to use the same name that is used in other DBs.

File: athenacli/packages/special/iocommands.py [hunk 2]
```diff
@@ -219,6 +219,34 @@ def subst_favorite_query_args(query, args):
...
+def execute_file_query(cur, arg, **_):
+    filename = arg
+    if filename:
+        try:
+            with open(filename, encoding='utf-8') as f:
+                query = f.read()
+                for sql in sqlparse.split(query):
```
Comment (zzl0) [inline athenacli/packages/special/iocommands.py:229]: Remove the empty string around here?

File: athenacli/packages/special/iocommands.py (hunk 1 above)
Comment (sco11morgan) [reply athenacli/packages/special/iocommands.py:222]: will do

**Comment by sco11morgan** [comment]
Thanks for the feedback.  Will update in the next day.

**Comment by zzl0** [comment]
LGTM, thanks

**Pull request merged**